Kitodo/src/main/java/org/kitodo/production/process/TiffHeaderGenerator.java

```diff
@@ -69,38 +69,38 @@ public String generateTiffHeader(String tiffDefinition, String docType) throws P
                 tiffHeaderImageDescriptionBuilder.append(getTifHeaderType(docType));
             } else {
                 // otherwise, evaluate the token as a field name
-                tiffHeaderImageDescriptionBuilder.append(evaluateAdditionalFields(token));
-
+                tiffHeaderImageDescriptionBuilder.append(evaluateAdditionalDetailsRows(token));
             }
         }
         return reduceLengthOfTifHeaderImageDescription(tiffHeaderImageDescriptionBuilder.toString());
     }
 
-    private String evaluateAdditionalFields(String token) throws ProcessGenerationException {
+    private String evaluateAdditionalDetailsRows(String token) throws ProcessGenerationException {
```

Comment:
What does it evaluate?
It is somehow changing the AdditionalDetailsTab (e.g. by setting the additionalDetailsRow) but it is also creating a tiff header.